rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -126,18 +156,53 @@ impl FirmwareProcessor {
     fn process_mailbox_commands<'a>(
         soc_ifc: &mut SocIfc,
         mbox: &'a mut Mailbox,
+        env: &mut KatsEnv,
```

Comment:
Might be more future proof (for deferred measurements) to pass in the entire `&mut RomEnv` here, but we can defer that to a future PR.